test/e2e/specs/site-editor/block-list-panel-preference.spec.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Block list view', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test( 'Should open by default', async ( { page, pageUtils } ) => {
+		await pageUtils.visitSiteEditor( {
+			postId: 'emptytheme//index',
+			postType: 'wp_template',
+		} );
+
+		// Should display the Preview button.
+		await expect(
+			page.locator( '.edit-site-editor__list-view-panel' )
+		).not.toBeVisible();
+
+		// Turn on block list view by default.
+		await page.evaluate( () => {
+			window.wp.data
+				.dispatch( 'core/preferences' )
+				.set( 'core/edit-site', 'showListViewByDefault', true );
+		} );
+
+		await page.reload();
+
+		// Should display the Preview button.
+		await expect(
+			page.locator( '.edit-site-editor__list-view-panel' )
+		).toBeVisible();
+	} );
```

Comment:
@kevin940726 to check that my playwright test isn't a total mess 😄 

I couldn't get something like `'role=region[aria-label="List View"]'` working, hence the classname

Reply:
How about something like `'role=region[name="List View"i]'`?

Reply:
> How about something like 'role=region[name="List View"i]'?

Thanks, that works! 🍺

I was naively trying with `aria-label`